tests/foreman/api/test_host.py

```diff
@@ -1068,6 +1069,51 @@ def test_positive_read_enc_information(
         assert host_enc_parameters[param['name']] == param['value']
 
 
+@pytest.mark.e2e
+def test_positive_bootc_api_actions(target_sat, bootc_host, function_ak_with_cv, function_org):
+    """Register a bootc host and validate API information
+
+    :id: b94ab231-0dd8-4e47-a96b-972c5ee55f4d
+
+    :expectedresults: Upon registering a Bootc host, the API returns correct information across multiple endpoints
+
+    :CaseComponent:Hosts-Content
```

Comment:
Please confirm the component. I think this test should be part of Hosts component, as it just a registering host and validating its information and nothing related to content manipulation.
```suggestion
    :CaseComponent:Hosts
```

Reply:
we do not own the hosts component, it's possible this should be a new component, but if it's flagged as host we will not see this in our test results, depsite owning the feature and future development of that feature